source/slang/slang-emit-cuda.cpp

```diff
@@ -788,6 +788,29 @@ bool CUDASourceEmitter::tryEmitInstExprImpl(IRInst* inst, const EmitOpInfo& inOu
             m_writer->emit(")getOptiXRayPayloadPtr()");
             return true;
         }
+        case kIROp_GetOptiXHitAttribute:
+        {
+            auto typeToFetch = inst->getOperand(0);
+            auto idxInst = as<IRIntLit>(inst->getOperand(1));
+            IRIntegerValue idx = idxInst->getValue();
+            if (typeToFetch->getOp() == kIROp_FloatType) {
```

Comment:
This is a pedantic point but our coding standard has allman rather than K&R indentation style. So { should be on next line and tabbed to the if. https://github.com/shader-slang/slang/blob/master/docs/design/coding-conventions.md